src/http-client.js

```diff
@@ -3,13 +3,14 @@ import {Headers} from './headers';
 import {RequestBuilder} from './request-builder';
 import {HttpRequestMessage,createHttpRequestMessageProcessor} from './http-request-message';
 import {JSONPRequestMessage,createJSONPRequestMessageProcessor} from './jsonp-request-message';
+import { RequestMessageProcessor } from './request-message-processor';
 
-function trackRequestStart(client, processor){
+function trackRequestStart(client : HttpClient, processor : RequestMessageProcessor){
```

Comment:
These are private helper functions, so I don't think there's much value in annotating them.